includes/Customizer.php

```diff
@@ -0,0 +1,568 @@
+			]
+		);
+
+		$wp_customize->add_setting(
+			self::STORY_OPTION . '[show_stories]',
+			[
+				'default' => false,
+				'type'    => 'option',
+			]
+		);
+
+		$wp_customize->add_control(
+			self::STORY_OPTION . '[show_stories]',
+			[
+				'type'    => 'checkbox',
+				'section' => self::SECTION_SLUG,
+				'label'   => __( 'Show stories', 'web-stories' ),
+			]
+		);
+
+		$wp_customize->add_setting(
+			self::STORY_OPTION . '[view_type]',
+			[
+				'default' => $theme_support['view-type-default'],
+				'type'    => 'option',
+			]
+		);
+
+		$wp_customize->add_control(
+			self::STORY_OPTION . '[view_type]',
+			[
+				'section'         => self::SECTION_SLUG,
+				'label'           => __( 'Story view type', 'web-stories' ),
+				'type'            => 'select',
+				'choices'         => $this->get_view_type_choices( $theme_support['view-type'] ),
+				'active_callback' => function() {
+					return $this->is_option_enabled( 'show_stories' );
+				},
+			]
+		);
+
+		$wp_customize->add_setting(
+			self::STORY_OPTION . '[number_of_stories]',
+			[
+				'default'           => $theme_support['number-of-stories'],
+				'type'              => 'option',
+				'validate_callback' => [ $this, 'validate_number_of_stories' ],
+			]
+		);
+
+		$wp_customize->add_control(
+			self::STORY_OPTION . '[number_of_stories]',
+			[
+				'type'            => 'number',
+				'section'         => self::SECTION_SLUG,
+				'label'           => __( 'Number of stories', 'web-stories' ),
+				'input_attrs'     => [
+					'min' => 1,
+					'max' => 20,
+				],
+				'active_callback' => function() {
+					return $this->is_option_enabled( 'show_stories' );
+				},
+			]
+		);
+
+		$wp_customize->add_setting(
+			self::STORY_OPTION . '[number_of_columns]',
+			[
+				'default'           => $theme_support['grid-columns-default'],
+				'type'              => 'option',
+				'validate_callback' => [ $this, 'validate_number_of_columns' ],
+			]
+		);
+
+		$wp_customize->add_control(
+			self::STORY_OPTION . '[number_of_columns]',
+			[
+				'type'            => 'number',
+				'section'         => self::SECTION_SLUG,
+				'label'           => __( 'Number of columns', 'web-stories' ),
+				'input_attrs'     => [
+					'min' => 1,
+					'max' => 4,
+				],
+				'active_callback' => function() {
+					return ( $this->is_option_enabled( 'show_stories' ) && $this->is_view_type( 'grid' ) );
+				},
+			]
+		);
+
+		$wp_customize->add_setting(
+			self::STORY_OPTION . '[order]',
+			[
+				'default' => $theme_support['order-default'],
+				'type'    => 'option',
+			]
+		);
+
+		$wp_customize->add_control(
+			self::STORY_OPTION . '[order]',
+			[
+				'section'         => self::SECTION_SLUG,
+				'label'           => __( 'Order by', 'web-stories' ),
+				'type'            => 'select',
+				'choices'         => $this->get_order_choices( $theme_support['order'] ),
+				'active_callback' => function() {
+					return $this->is_option_enabled( 'show_stories' );
+				},
+			]
+		);
```

Comment:
These should be broken into two fields as noted elsewhere. Order or order by. Having this as one field feels wrong. You agree @swissspidy ?